src/main/java/uk/gov/hmcts/reform/prl/models/dto/payment/PaymentStatusForCitizen.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.prl.models.dto.payment;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PaymentStatusForCitizen {
```

Comment:
@SangeethaPonnusamy Can we rename this to "PaymentStatusResponse" please to make it generic.

Reply:
@yogendraU Please comment if this is fine ?